multiworld/envs/mujoco/sawyer_xyz/sawyer_push_and_reach_env.py

```diff
@@ -212,9 +212,8 @@ def _set_puck_xy(self, pos):
     def reset_model(self):
         self._reset_hand()
         goal = self.sample_goal()
-        self._state_goal = goal['state_desired_goal']
+        self.set_goal(goal)
         self._set_goal_marker(self._state_goal)
```

Comment:
You could/should probably move `_set_goal_marker` to inside `set_goal` now

Reply:
ok will do!